core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -174,6 +174,35 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     getOption(key).getOrElse(defaultValue)
   }
 
+  /** Get a time parameter as seconds; throws a NoSuchElementException if it's not set. If no 
+    * suffix is provided then seconds are assumed. 
+    */
+  def getTimeAsSec(key: String): Long = {
+    Utils.timeStringAsSec(get(key))
```

Comment:
I personally still prefer these to be `*AsSeconds` so it looks a little more complete. Note that this will be new public API in Spark 1.4.